notebook/notebookapp.py

```diff
@@ -239,16 +239,21 @@ def init_handlers(self, settings):
         handlers.extend(load_handlers('services.security.handlers'))
         
         # BEGIN HARDCODED WIDGETS HACK
+        widgets = None
         try:
-            import ipywidgets
+            widgets = __import__('jupyter-js-widgets-nbextension')
```

Comment:
if you have `jupyter-js-widgets-nbextension` installed, does `import jupyter_js_widgets_nbextension as widets` works ?